content/terraform-plugin-framework/v1.15.x/docs/plugin/framework/migrating/resources/timeouts.mdx

```diff
@@ -97,18 +145,22 @@ func (e *exampleResource) Create(ctx context.Context, req resource.CreateRequest
 	resp.Diagnostics.Append(diags...)
 ```
 
-Modify the `exampleResourceData` model to include a field for timeouts using a [`timeouts.Value`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework-timeouts/resource/timeouts#Value) type.
+Modify the `exampleResourceData` model to include a field for timeouts using a
+[`timeouts.Value`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework-timeouts/resource/timeouts#Value)
+type.
 
 ```go
 type exampleResourceData struct {
     /* ... */
     Timeouts    timeouts.Value `tfsdk:"timeouts"`
 ```
 
-## Accessing Timeouts in CRUD Functions
+## Accessing timeouts in CRUD functions
 
-Once the model has been populated with the config, state or plan the duration of the timeout can be accessed by calling
-the appropriate helper function (e.g., [`timeouts.Create`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework-timeouts/resource/timeouts#Value.Create)) and then used to configure timeout behaviour, for instance:
+Once the model has been populated with the config, state or plan the duration of
+the timeout can be accessed by calling the appropriate helper function (e.g.,
+[`timeouts.Create`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework-timeouts/resource/timeouts#Value.Create))
+and then used to configure timeout behaviour, for instance:
```

Comment:
```suggestion
After populating the model with the configuration, state, or plan, Terraform can access the duration of the timeout by calling the appropriate helper function, such as [`timeouts.Create`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework-timeouts/resource/timeouts#Value.Create), so that it can configure timeout behavior. For example:
```
I didn't know how to explain the example 😳 so I just left the two word intro. Ideally, we'd explain it in a little more detail.